app/src/main/java/com/theappbusiness/kc/controllers/categories/CategoriesActivity.java

```diff
@@ -1,48 +1,38 @@
-package com.theappbusiness.kc.view.categories;
+package com.theappbusiness.kc.controllers.categories;
 
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
 
 import com.theappbusiness.kc.KcApp;
 import com.theappbusiness.kc.R;
-import com.theappbusiness.kc.data.Database;
 import com.theappbusiness.kc.di.components.CategoriesComponent;
 import com.theappbusiness.kc.di.modules.CategoriesModule;
-import com.theappbusiness.kc.di.qualifiers.ForCategories;
-import com.theappbusiness.kc.io.CategoriesController;
-import com.theappbusiness.kc.io.CategoriesManagerImpl;
-import com.theappbusiness.kc.io.KcObservables;
-import com.theappbusiness.kc.io.Manager;
 import com.theappbusiness.kc.model.Category;
-import com.theappbusiness.kc.service.BackendOperations;
+import com.theappbusiness.kc.network.categories.CategoriesNetworkManager;
+import com.theappbusiness.kc.persistence.Database;
+import com.theappbusiness.kc.view.categories.CategoriesPresentation;
 
 import java.util.List;
 
 import javax.inject.Inject;
 
-import rx.Observable;
-
 /**
  * Acts as {@link CategoriesController}. Delegates presentations tasks to {@link
- * CategoriesPresentation} and network tasks to {@link CategoriesManagerImpl}
+ * CategoriesPresentation} and com.theappbusiness.kc.network tasks to {@link CategoriesManagerImpl}
  */
 public class CategoriesActivity extends AppCompatActivity implements CategoriesController {
 
     private CategoriesComponent mComponent;
 
     @Inject
-    CategoriesPresentation categoriesPresentation;
+    CategoriesPresentation mPresentation;
 
     @Inject
-    BackendOperations mBackendOperations;
+    CategoriesNetworkManager mNetworkManager;
 
```

Comment:
I left it there , but I don't think we will need any db , at least we don't need it atm